jsonschema-core.xml

```diff
@@ -2141,11 +2141,11 @@
                             "additionalProperties" and itself
                         </t>
                         <t>
-                            "additionalItems", whose behavior is defined in terms of "items"
+                            "items", whose behavior is defined in terms of "prefixItems"
                         </t>
                         <t>
                             "unevaluatedItems", whose behavior is defined in terms of annotations
-                            from "items", "additionalItems" and itself
+                            from "prefixItems", "items", "contains", and itself
```

Comment:
I still firmly disagree with `contain` influencing `unevaluatedItems`.  Can we separate the `items` split and the `contains` issue into two PRs, please?

Reply:
I've been externally convinced, though I still don't like the names.  I'll open a new issue for the naming of things.

Reply:
@gregsdennis we spent an epic number of comments debating the naming, with participation from the OpenAPI steering committee as well.  In the end, @Relequestual made a call and that's what I implemented.

I do apologize for not realizing the `contains` things wasn't signed off on, but `prefixItems` was [signed off in this comment](https://github.com/json-schema-org/json-schema-spec/issues/864#issuecomment-598313804) and then after more comments, was [reaffirmed in this comment](https://github.com/json-schema-org/json-schema-spec/issues/864#issuecomment-601147496).